bin/fin

```diff
@@ -4831,7 +4839,8 @@ case "$1" in
 		elif [[ "$1" == "--tools" ]]; then
 			update_tools
 		# TODO: "--config" deprecated, remove at some point
-		elif [[ "$1" == "--config" ]] || [[ "$1" == "--stacks" ]]; then
+		elif [[ "$1" == "--config" ]] || [[ "$1" == "--stack" ]] || [[ "$1" == "--stacks" ]]; then
```

Comment:
Since this pulls all stack files, lets keep it as `--stacks` and drop other options now.
This is an internal argument and it is not mentioned in help or docs.